module/VuFind/src/VuFind/Db/Service/TagService.php

```diff
@@ -1062,8 +1063,8 @@ public function getListTagsForUser(
     /**
      * Get tags assigned to a user list.
      *
-     * @param int    $listId List ID
-     * @param string $userId User ID to look up (null for no filter).
+     * @param int|UserList $listId List ID
+     * @param string|User  $userId User ID to look up (null for no filter).
```

Comment:
Any idea why this is documented as string rather than int? And should we consider renaming the parameter to $user since in future it's probably more likely to be an object than an ID, and that could lead to confusion?